front/app/containers/MobileNavbar/FullMobileNavMenuItem.tsx

```diff
@@ -35,26 +39,36 @@ const StyledLink = styled(Link)`
 
 interface Props {
   linkTo: string;
-  navigationItemTitle: Multiloc;
+  navigationItemTitle?: Multiloc;
+  navigationItemTitleCopy?: MessageDescriptor;
```

Comment:
I tried to combine two types as described here https://stackoverflow.com/questions/40510611/typescript-interface-require-one-of-two-properties-to-exist, but destructuring doesn't work in this case saying
```
Property 'navigationItemTitle' does not exist on type '(MultilocProps | CopyProps) & InjectedIntlProps'
```